tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>Deltas from TF docs reviewers guidance when the QAT docs were reviewed:</p><p> </p><p>1) "to determine whether you want to use" -&gt; "to determine if you should use it" </p><p>2) "your use case" -&gt; "your use case (beyond fully clustering a model with 32 clusters)"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>